ui/src/styles/_cards.scss

```diff
@@ -29,7 +29,12 @@
       --buttonHoverColor: var(--primaryColor);
     }
   }
-
+  &--white {
+    padding: var(--largestSpace);
+    box-shadow: 6px 20px 32px rgba(0, 0, 0, 0.2);
+    border-radius: 16px;
+    --cardBackground: var(--white);
+  }
```

Comment:
by default cards should be white so this class probably isn't necessary. Also the other properties added in this class dont have to do with the color white.
- Padding should have its own class, most likely not associated with the card, probably in layouts.scss like padding--largest
- box shadow should be a pre-definled elevation, not something new. we already have elevation classes for cards that can be used
- according to the mockups, the border radius on this is the same as all other cards so a new property shouldn't be made. if other card radii are wrong then this should be updated in vars